NEWS

```diff
@@ -1,3 +1,6 @@
+1.7.3:
+* replaced deprecated path API uses
```

Comment:
- [ ] add more information about how `path.py>=`[version] obsoleted the methods previously used
- [ ] also add the new minimum version of `path.py` that supports these methods to `pyproject.toml`

Alternatively, could switch to Python's builtin `pathlib`.